code/_globalvars/lists/loadout_categories.dm

```diff
@@ -0,0 +1,13 @@
+GLOBAL_LIST_INIT(loadout_categories, list(
+	CATEGORY_BACKPACK = list(SUBCATEGORY_BACKPACK_GENERAL, SUBCATEGORY_BACKPACK_TOYS),
+	CATEGORY_NECK = list(SUBCATEGORY_NECK_GENERAL, SUBCATEGORY_NECK_TIE, SUBCATEGORY_NECK_SCARVES),
+	CATEGORY_MASK = list("NOSUBCATEGORY"),
```

Comment:
make this a define (NOSUBCATEGORY) please thanks

Reply:
or just an empty list

Reply:
ill define NOSUBCATEGORY because the way I have it working is having NOSUBCATEGORY be a single subcategory for the purpose of for loops, except it is a specific edge-case where the subcategory won't be shown and by default everything is in it for that main category